lc-hax/Scripts/Helpers/Enemies.cs

```diff
@@ -29,4 +77,19 @@ internal static partial class Helper {
 
         return null;
     }
+    internal static void MouthDogChasePlayer(this MouthDogAI instance, PlayerControllerB player) {
+        if (instance == null) return;
+        if (player == null) return;
+        if (instance.currentBehaviourStateIndex is 0 or 1) {
+            _ = instance.Reflect().InvokeInternalMethod("ChaseLocalPlayer");
+        }
+        else {
+            if (instance.currentBehaviourStateIndex != 2 || instance.Reflect().GetInternalField<bool>("inLunge"))
+                return;
+            instance.transform.LookAt(player.transform.position);
+            instance.transform.localEulerAngles = new Vector3(0.0f, player.transform.eulerAngles.y, 0.0f);
+            _ = instance.Reflect().SetInternalField("inLunge", true);
+            _ = instance.Reflect().InvokeInternalMethod("EnterLunge");
+        }
+    }
```

Comment:
Make this inline. No point abstracting something we will never use. Even if we do want to use this, it doesn't follow SRP anyways.

Reply:
is being used in god mode , to make the dog have a controlled interaction with the local player.

Reply:
ya so just write it inline there. don’t add this to helper.

Reply:
you are polluting the helper namespace

Reply:
All done, moved it to the patch directly.